debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java

```diff
@@ -238,10 +236,20 @@ private boolean isWhitespace(byte c) {
 
     private void doProcessMessage(ReplicationMessageProcessor processor, TypeRegistry typeRegistry, byte[] content, boolean lastMessage)
             throws IOException, SQLException, InterruptedException {
-        final Document change = DocumentReader.floatNumbersAsTextReader().read(content);
+        if(content != null) {
+            final Document change = DocumentReader.floatNumbersAsTextReader().read(content);
+            LOGGER.trace("Change arrived for decoding {}", change);
+            processor.process(new Wal2JsonReplicationMessage(txId, commitTime, change, containsMetadata, lastMessage, typeRegistry));
+        }
+        else
+        {
```

Comment:
Fails with checkstyle, the left brace should go to the previous line which will be then
```
else {
```